python/lsst/ts/adamSensors/adamSensorsCSC.py

```diff
@@ -44,9 +40,27 @@ async def begin_start(self, data):
         self.cmd_start.ack_in_progress(data, timeout=self.start_timeout)
         await super().begin_start(data)
         self.adam = AdamModel(self.log, simulation_mode=self.simulation_mode)
-        self.adam.connect(self.params.ip, self.params.port)
+        try:
+            await self.adam.connect(self.config.adam_ip, self.config.adam_port)
+        except ConnectionException:
+            raise RuntimeError(f"Unable to connect to modbus device at {self.config.adam_ip}:\
```

Comment:
instead of using "\" do:

```
raise RuntimeError(
    f"Unable to connect to modbus device at " 
    f"{self.config.adam_ip}:{self.config.adam_port}.")
```